.github/workflows/on_pull_request.yml

```diff
@@ -94,8 +94,8 @@ jobs:
     uses: ./.github/workflows/reusable_test_wheels.yml
     with:
       CONCURRENCY: pr-${{ github.event.pull_request.number }}
-      PLATFORM: linux
-      WHEEL_ARTIFACT_NAME: "linux-wheel-fast"
```

Comment:
A documented intention and motivation would be useful here and elsewhere

(i.e. why do we only run the minimum wheel test on arm?)

Reply:
That was a temporary hack to get the workflow to run. That said, yes I wish that this choice was previously documented.